v2/cdc-parent/cdc-change-applier/src/main/java/com/google/cloud/dataflow/cdc/applier/CdcToBigQueryChangeApplierPipeline.java

```diff
@@ -93,11 +93,12 @@
     String getChangeLogDataset();
     void setChangeLogDataset(String dataset);
 
-    @Description("The BigQuery dataset where the Replica tables are to be kept.")
+    @Description("The BigQuery dataset where the Replica tables are to be kept. If empty replica operation is skipped.")
     String getReplicaDataset();
     void setReplicaDataset(String dataset);
 
-    @Description("How often the pipeline will issue updates to the BigQuery replica table.")
+    @Description("How often the pipeline will issue updates to the BigQuery replica table. Default Hourly")
+    @Default.Integer(1440)
```

Comment:
Can we avoid changing the default for now? I'd rather we don't change this behavior without consulting with other potential users.